packages/hooks/src/useWalletAccountsList.ts

```diff
@@ -93,7 +93,7 @@ const useWalletAccountsList = () => {
                 /** Landing company shortcode the account belongs to. */
                 landing_company_name: wallet.landing_company_name?.replace('maltainvest', 'malta'),
                 /** Indicating whether the wallet is a maltainvest wallet. */
-                is_malta_wallet: wallet.landing_company_name === 'malta',
+                is_malta_wallet: wallet.landing_company_name === 'maltainvest',
```

Comment:
Note1: changed to `maltainvest` because `replace` function doesn't mutate original string and it's still `maltainvest`, not `malta`. Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Note2: we don't use `Jurisdiction` object from `shared` package because hooks package doesn't have this package as dependency